SingularityUI/app/assets/index.mustache

```diff
@@ -53,7 +53,8 @@
             authCookieName: "{{{ authCookieName }}}",
             authTokenKey: "{{{ authTokenKey }}}",
             quickLinks: {{{quickLinks}}},
-            navTitleLinks: {{{navTitleLinks}}}
+            navTitleLinks: {{{navTitleLinks}}},
+            lessTerminalPort: {{{lessTerminalPort}}}
```

Comment:
quick note for this. To fully wire it up to Singularity backend, we'll have to put something in maybe the [UI Config](https://github.com/HubSpot/Singularity/blob/master/SingularityService/src/main/java/com/hubspot/singularity/config/UIConfiguration.java) that is then wired to the [IndexView](https://github.com/HubSpot/Singularity/blob/master/SingularityService/src/main/java/com/hubspot/singularity/views/IndexView.java). That will wire the value through from the Singularity config yaml through the the frontend config